src/sentry/web/frontend/oauth_authorize.py

```diff
@@ -286,7 +325,18 @@ def _logged_out_post(
         response = super().post(request, application=application, **kwargs)
         # once they login, bind their user ID
         if request.user.is_authenticated:
-            request.session["oa2"]["uid"] = request.user.id
+            # Regenerate session to prevent session fixation attacks
```

Comment:
not totally convinced this is doing anything, but maybve i misunderstand the attack vector?